client/src/cmdhfcipurse.c

```diff
@@ -651,10 +651,10 @@ static int CmdHFCipurseDeleteFile(const char *Cmd) {
 
     if (verbose) {
         PrintAndLogEx(INFO, "File id " _YELLOW_("%x") " key id " _YELLOW_("%d") " key " _YELLOW_("%s")
-            , fileId
-            , keyId
-            , sprint_hex(key, CIPURSE_AES_KEY_LENGTH)
-            );
+                      , fileId
+                      , keyId
+                      , sprint_hex(key, CIPURSE_AES_KEY_LENGTH)
+                     );
```

Comment:
looks like you somehow didn't base your PR on latest source and got unrelated stuff in your PR